### DISCUSSION TITLE
Cell's alarm ?

### DISCUSSION THREAD

**Discussion by tlamoureux24** [syssi/esphome-seplos-bms#100] [Q&A] [answered]
Hello, first of all thank you for this program esphome it works at the top!! I have a small question, is it possible to integrate the alarm sensors of the cells into the program? I'm thinking mainly of high voltage alarm and over voltage protection sensors. 
Thank you :)

**Comment by syssi** [comment, marked as answer]
It's not implemented yet but it's the goal of #6. Please subscribe this issue and provide some testing if you have the chance!
